testsuite/unit/src/burst_buffer/dws-test.lua

```diff
@@ -1066,23 +925,62 @@ describe("Slurm API", function()
 	it("slurm_bb_setup through all other states", function()
 		call_bb_setup()
 		call_bb_state("DataIn")
+		call_bb_get_status("DataIn", "Completed")
 		call_bb_state("PreRun")
 		call_bb_state("PostRun")
 		call_bb_state("DataOut")
 		call_bb_teardown()
 	end)
 
+	insulate("reports driver error(s)", function()
```

Comment:
When I experimented with insulate() a few months ago, I found that it has a suprisingly narrow purpose.  It probably does nothing special for us here.  If you switch it to context() does it break this code block or affect the next context()?

Reply:
Yeah, I was hoping `insulate` would do more. `context` works just fine in this case. I'll replace it.

Reply:
Here's one place where it does something useful:
https://github.com/DataWorkflowServices/dws-slurm-bb-plugin/blob/main/testsuite/unit/src/burst_buffer/dws-test.lua#L1103-L1112